R/proc.cdf.R

```diff
@@ -43,8 +43,30 @@ load_data <- function(filename,
   return(raw.prof)
 }
 
-#' Process file and return the profile.
+#' Filter noise and detect peaks from LC/MS data in CDF format
```

Comment:
This actually not exclusive for cdf format - likely the function should be renamed in the future.

Reply:
See #79